tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -86,6 +87,29 @@ describe('package/', () => {
     })
   })
 
+  describe('GOV.UK Prototype Kit', () => {
+    let listingPackagePrototype
+
+    beforeAll(async () => {
+      listingPackagePrototype = await getListing(join(configPaths.package, 'govuk-prototype-kit'))
+    })
+
+    it('should have JSON config', () => {
+      expect(listingPackage)
+        .toEqual(expect.arrayContaining(['govuk-prototype-kit.config.json']))
+    })
+
+    it('should have init.js', () => {
+      expect(listingPackagePrototype)
+        .toEqual(expect.arrayContaining(['init.js']))
+    })
+
+    it('should have init.scss', () => {
+      expect(listingPackagePrototype)
+        .toEqual(expect.arrayContaining(['init.scss']))
+    })
+  })
+
```

Comment:
These are already covered by the 'contains the expected files' test which:

- looks for a matching file for everything in the `src/` directory (apart from the excluded files), and so already expects corresponding files in the package for `govuk-prototype-kit/init.js` and `govuk-prototype-kit/init.scss`
- includes `govuk-prototype-kit.config.json` in [the list of 'extra' files](https://github.com/alphagov/govuk-frontend/blob/4c2a1503f2a54f0b7333bf203e6bad5735e1afbc/tasks/gulp/__tests__/after-build-package.test.js#L68)

Reply:
Fab, just me being overly cautious. I'll remove 👍

Reply:
@36degrees These are gone now

I've also swapped the "already present" `govuk-prototype-kit.config.json` with mapped replacement:

```mjs
// Replaces GOV.UK Prototype kit config with JSON
.flatMap(mapPathTo(['**/govuk-prototype-kit.config.mjs'], ({ dir: requirePath, name }) => [
  join(requirePath, '../', `${name}.json`)
]))
```

Similar to the YAML file transform, it shows the config is replaced with JSON a directory up (same name)